content/docs/guides/opentelemetry.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Ingesting OpenTelemetry Metrics
+---
+
+# Ingest Native OpenTelemetry Metrics to Prometheus
+
+Prometheus supports ingesting native OpenTelemetry Metrics from release 2.47 onwards. Before this change, Prometheus was not supporting ingesting the OpenTelemetry Metrics natively, there was a way to convert the OpenTelemetry Metrics into Prometheus Metrics format either via Prometheus Remote Write Exporter.
+
+In this guide, we will learn about
+
+- Configuration changes required in Prometheus setup to enable the ingestion of OpenTelemetry Metrics.
+- Setting up a Python application with OpenTelemetry SDK.
+- Emitting OpenTelemetry metrics via auto-instrumentation.
+- Ingesting those metrics in Prometheus natively.
```

Comment:
There is a lot happening here. It's nice to have a complete example, but for the purpose of demonstrating Prometheus, isn't this a bit too much that is unrelated to the topic at hand? 

Setting up flask app and auto-instrumentations feels like out of scope for basic tutorial here? I believe a basic cli that pushes manually collected metrics once should be a more appropriate level. Then linking to opentelemetry tutorials for more advanced examples.

Especially considering this could be read by someone who doesn't come from Python background.

What do others think?

Reply:
@gouthamve Thoughts?